cluster/examples/kubernetes/ceph/upgrade-from-v1.3-crds.yaml

```diff
@@ -356,3 +356,266 @@ spec:
   version: v1
   subresources:
     status: {}
+---
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Should this file be included in a separate PR so we can backport the rest to 1.3?

Reply:
The rest is already in 1.3 with https://github.com/rook/rook/pull/5780

Reply:
of course!